Makefile

```diff
@@ -10,49 +10,67 @@
 ##########################################################################
 
 MODULE_big = sqlite_fdw
-OBJS = connection.o option.o deparse.o sqlite_query.o sqlite_fdw.o sqlite_data_norm.o
+OBJS = connection.o option.o deparse.o sqlite_query.o sqlite_fdw.o sqlite_data_norm.o sqlite_gis.o
```

Comment:
No, @son-phamngoc . `sqlite_gis.c` contents obligatory string arrays like [here](https://github.com/mkgrgis/sqlite_fdw/blob/67fead455e15f66d76f16caad372d84307979635/sqlite_gis.c#L43), used for warning about GIS data types in case of compilation without GIS support. I think we should give user warning in case of potentially GIS formally data types in SQLite.

Reply:
Okay. Please keep this configuration.